src/main/webapp/components/workspace/workspace.js

```diff
@@ -212,6 +242,12 @@ export default () => {
                     })
                   }}
                 />
+                <Filters
+                  filters={filters}
```

Comment:
I think it would be better to just pass the onChange to the Filters component, and call onChange with the filters stored locally on that component, or [ ] when the enabled/disabled switch is toggled - should simplify `onFilterChange` and the `onSearchQuery` a bit